### PR TITLE
Updating pni about

### LINKED ISSUE #3800: PNI: Implement About Page additions
Design ticket https://github.com/mozilla/foundation.mozilla.org/issues/3619

## List of Requirements
- Update and add new about pages
- Keep fonts, layout, and general styling the same as our current about pages
- remove accordions (mainly for _How to Use this Guide_)

## Copy

- [x] [How to Use This Guide](https://docs.google.com/document/d/12QuPQ_RXkmLlAlhCcPTX50-Nup2JskAIwC5ntcn8R4Q/edit)
- [x] [Why We Made This Guide](https://docs.google.com/document/d/1rMo-TNMcHkgYOs2JSP8UxpW6AHdrx3N2odbL-uRP1Xs/edit)
- [x] [Minimum Security Guidelines Explained](https://docs.google.com/document/d/1Y49QM9Yb-BWI7IhtwQUdXMWS6CQoIy8WN6j8Ox_rZSE/edit#)
- [x] [About our Methodology](https://docs.google.com/document/d/1oaPPgFRlA1_zfpd5tPPe8THC20iFnmSYBvSvFppkmjA/edit)
- [x] [Press](https://docs.google.com/document/d/153YTZ2edXYxMUtFAXyX34d7-WlKCW4xC9mYQd7i36JA/edit#heading=h.2ws9vo67f5s)

### PR DESCRIPTION
**Pull request by youriwims**
Closes #3800
Related PRs/issues #3863

View updates 👉 https://foundation-mofostaging-pr-3864.herokuapp.com/en/privacynotincluded/about/

### CODE REVIEW

**Comment by youriwims** [comment]
@kristinashu can you also check to make sure the urls are okay? For example, currently I have `about/methodology`, would you prefer `about/our-methodology`? Let me know if you have no preference.

**Comment by kristinashu** [comment]
Links work for me!

**Comment by Pomax** [comment]
Something about this PR is breaking the python test run (https://travis-ci.org/mozilla/foundation.mozilla.org/jobs/611172904?utm_medium=notification&utm_source=github_status)... so I guess we need to figure out what that something is =x

File: network-api/networkapi/buyersguide/urls.py [hunk 1]
```diff
@@ -4,9 +4,12 @@
 
 urlpatterns = [
     url(r'^$', views.buyersguide_home, name='buyersguide-home'),
-    url(r'^about/contact', views.contact_view, name='contact-view'),
-    url(r'^about/why', views.why_view, name='why-view'),
-    url(r'^about/', views.about_view, name='about-view'),
+    url(r'^about/press', views.bg_about_page("press"), name='press-view'),
+    url(r'^about/contact', views.bg_about_page("contact"), name='contact-view'),
+    url(r'^about/methodology', views.bg_about_page("methodology"), name='methodology-view'),
+    url(r'^about/meets-minimum-security-standards', views.bg_about_page("minimum_security"), name='explained-view'),
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/urls.py]: `name` doesn't really match the url here

File: network-api/networkapi/buyersguide/urls.py [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
+    url(r'^about/why', views.bg_about_page("why_we_made"), name='why-view'),
+    url(r'^about/', views.bg_about_page("how_to_use"), name='about-view'),
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/urls.py]: same here, `name` feels a little off

**Pull request merged**